response_operations_ui/views/surveys.py

```diff
@@ -38,7 +38,33 @@ def view_collection_exercise(short_name, period):
 
 
 @app.route('/surveys/<short_name>/<period>', methods=['POST'])
-def upload_collection_instrument(short_name, period):
+def upload(short_name, period):
+    if 'load-sample' in request.form:
+        return _upload_sample(short_name, period)
+    else:
+        return _upload_collection_instrument(short_name, period)
+
+
+def _upload_sample(short_name, period):
+    error = _validate_sample()
+    sample = False
+    total_businesses = request.form.get('sample-businesses')
+    total_ci = request.form.get('sample-collection-instruments')
+
+    ce_details = collection_exercise_controllers.get_collection_exercise(short_name, period)
```

Comment:
Currently the [PR in backstage](https://github.com/ONSdigital/ras-backstage/pull/29) expects the same URL structure ('/<short_name>/<period>') for uploading collection instruments as samples. Perhaps we don't need to get the relevant collection exercise here?